connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTaskContext.java

```diff
@@ -109,4 +111,13 @@ public void resume(TopicPartition... partitions) {
     public Set<TopicPartition> pausedPartitions() {
         return pausedPartitions;
     }
+
+    @Override
+    public void disableOffsetCommit() {
```

Comment:
Yeah, makes sense. I was mainly wondering if it would make sense to add an `enableOffsetCommit`, but it seems there's not really any use case for it.